nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -395,6 +399,29 @@ def simulator_run_main(self):
             run_status = 1
         return run_status
 
+    def split_gpus(self, gpus) -> ([str], bool):
+        gpus = gpus.replace(" ", "")
+        if not (gpus.startswith("[") and gpus.endswith("]")):
```

Comment:
Maybe we could use a different notation: instead of [1,2,3], we could use 1:2:3. This is much easier to parse - you don't need to worry about pairing up brackets. Just do split on ":".

Reply:
I did think of using format 1:2:3 first. Only reason I didn't adopt it is this may looks like not so natural for a list / group of GPUs. Definitely the parsing for that would be much easier. -- But, maybe 1:2:3 format is also acceptable.

Reply:
The bracket syntax is fine. But we should make it backward compatible.

Reply:
Made it backward compatible, accepts "[0,1],3", or "0,1,2".